modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -192,15 +193,63 @@ function validateInput($values)
     $db         =& Database::singleton();
     $pscid      = (isset($values['PSCID']) ? $values['PSCID'] : null);
     $visitLabel = (isset($values['visitLabel']) ? $values['visitLabel'] : null);
+    $centerID   = (isset($values['centerID']) ? $values['centerID'] : null);
     $result     = [
                    'PSCID'             => $pscid,
                    'visit'             => $visitLabel,
+                   'centerID'          => $centerID,
                    'candID'            => null,
                    'sessionID'         => null,
                    'isValidSubmission' => true,
                    'invalidMessage'    => null,
                   ];
 
+    if (isset($result['PSCID'], $result['centerID'])) {
+        $valid_center = $db->pselectOne(
+            "
+            SELECT
+                CenterID = :center_id
+            FROM
+                candidate
+            WHERE
+                PSCID = :psc_id
+        ",
+            array(
+             "center_id" => $result['centerID'],
+             "psc_id"    => $result['PSCID'],
+            )
+        );
+        if (!$valid_center) {
+            $valid_center = $db->pselectOne(
+                "
+                SELECT
+                    EXISTS (
+                        SELECT
+                            *
+                        FROM
+                            session s
+                        JOIN
+                            candidate c
+                        ON
+                            c.CandID = s.CandID
+                        WHERE
+                            s.CenterID = :center_id AND
+                            c.PSCID = :psc_id
+                    )
+            ",
+                array(
+                 "center_id" => $result['centerID'],
+                 "psc_id"    => $result['PSCID'],
+                )
+            );
+        }
+        if (!$valid_center) {
+            $message = "PSCID and Center ID do not match a valid session!";
```

Comment:
I will go do that rebase but I certainly hope it does not involve string concatenations. Because that is in no way clean >;(